src/components/Game/DeadScreen.tsx

```diff
@@ -19,9 +19,9 @@ function DeadScreen(): JSX.Element {
     return (
         <>
             <Contrast>
-                <div className="flex flex-col items-center w-full">
+                <div className="flex flex-col items-center w-full min-h-full">
                     <Heading1
-                        className="text-coral mb-10 text-55"
+                        className="text-coral lg:mb-10 text-55 sm:mb-5"
```

Comment:
Det bør være en "default" som gjelder for alle frem til et visst punkt. Og sm er ikke nødvendig. Så bytt gjerne sm:, med bare en default foran slik:

"text-coral text-55 mb-5 lg:mb-10 "

Reply:
👍